pkg/ccn-lite/Makefile

```diff
@@ -5,7 +5,7 @@ PKG_LICENSE=ISC
 
 .PHONY: all
 
-export RIOT_CFLAGS = $(CFLAGS) $(INCLUDES)
+RIOT_CFLAGS := $(INCLUDES)
```

Comment:
This should be probably be renamed to `RIOT_INCLUDES` to reflect its contents.

Reply:
mhm, those are still `CFLAGS` passed to the C compiler

Reply:
hmh, I can accept that. Still need someone else to review the newlib part, any suggestions who?